### PR TITLE
Divider

### PR DESCRIPTION
**Pull request by effe-megna**
## Description

This pull request add the Divider component to the library

#### Horizontal example:
Screenshot 2020-07-08 at 15 53 18

#### Text without heading example
Screenshot 2020-07-08 at 15 52 42

### Divider with title
Screenshot 2020-07-08 at 15 56 10

## Action Items

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] This PR is as small as it can be, and only implements on thing (not many)
- [x] I have manually tested my feature
- [ ] I have written a "good" amount of tests (including visual and unit tests)
- [x] Vertical type missing ([ant reference](https://ant.design/components/divider/#components-divider-demo-vertical))

### CODE REVIEW

File: src/Ant/Divider.elm [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        content =
+            case textStyle of
+                Plain ->
+                    Text.text label
+                        |> Text.toHtml
+
+                Heading ->
+                    {- missing functionality -}
+                    Text.text label
+                        |> Text.toHtml
```
[the remark this answers is not in the document]
Comment (effe-megna) [reply src/Ant/Divider.elm:131]: Yes, I will add a new level for this case

**Comment by supermacro** [comment]
Hey @effe-megna ... I pushed a commit that I think broke the `visual_tests` CI check. Going to try and figure out what's going on and push a fix to unblock you.

**Comment by supermacro** [comment]
I reverted some changes from `master` (in retrospect, that was not a smart thing to do), which means your branch currently has some commits that don't exist anymore.

You can fix this by running:

```
git reset aa9efb6d05ac4577b79f15b9f3482e4ddbae87d2
```

`aa9efb6d05ac4577b79f15b9f3482e4ddbae87d2` represents the most recent commit that you want to keep. 

Then you can do `git reset --hard` to delete all those commits that are no longer in `master`. 

Sorry about that again 🙈

**Comment by supermacro** [comment]
If you're up for the challenge you can also do an interactive rebase to remove `cda18848efe637baafe28c23015f3b485e373c49` (which is the source of the issue). My previous comment would prevent a few irrelevant commits from entering your PR and also `master`, but it doesn't solve the problem.  

https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history

**Comment by supermacro** [comment]
Actually .. i just had another thought. Don't worry about deleting commits. Once we merge I can resolve these issues. It's a lot easier that way.

So for this PR, we'll ignore the "visual_tests" ci check. It's not a big deal right now anyways.

**Comment by effe-megna** [comment]
Hey @supermacro, commits apart 😄 , I ended up with 3 visual tests and implemented the last 2 things, Vertical layout and Heading style (adding H5 in Typography). 

Let me know if something is missing

**Comment by supermacro** [comment]
I'll publish a new version this weekend or next week.

**Comment by supermacro** [comment]
It's live https://elm-antd.netlify.app/components/divider

**Pull request merged**